packages/pyright-internal/src/languageService/hoverProvider.ts

```diff
@@ -294,7 +302,7 @@ export class HoverProvider {
 
             case DeclarationType.Alias: {
                 this._addResultsPart(parts, '(module) ' + node.value, /* python */ true);
```

Comment:
change _addMouduleDocString to _addModuleParts ? and share this._addResultsparts part as well?

Reply:
Yeah good idea. No need to have dupe code.